src/app/pages/MarginTradePage/types.ts

```diff
@@ -73,3 +74,99 @@ export enum TradeType {
   'SHORT' = 'short',
   'LONG' = 'long',
 }
+
+export type LoanEvent = {
+  id: string;
+  type: EventType;
+  trade: EventTrade[];
+  isOpen: boolean;
+  loanToken: {
+    id: string;
+  };
+  liquidates: EventLiquidates[];
+  realizedPnL: string;
+  nextRollover: number;
+  startTimestamp: number;
+  closeWithSwaps: EventCloseWithSwaps[];
+  collateralToken: {
+    id: string;
+  };
+  depositCollateral: EventDepositCollateral[];
+  closewithDeposits: EventCloseWithDeposit[];
```

Comment:
> I know this is not your fault but I needed to check the graph to make sure this is not a typo. Ask Betsy/Remy how difficult would it be to rename this property as it looks really strange.

Hey tiltom, which property do you mean? Do you mean that depositCollateral is singular and closeWithDeposits is plural?

Reply:
It's this PR: https://github.com/DistributedCollective/Sovryn-frontend/pull/2131/files. It has a README on how to use it and how to write queries with it

Reply:
(I may be wrong about this, I'm not totally sure how codegen works in this case)

Reply:
@BetsyBraddock - yes, extra "s" is wrong but also it should be `closeWithDeposit` and not `closewithDeposit`.

Reply:
Thanks. It's closeWithDeposit on the graph, I think the typo is here. But yes about the extra "s" and the other typo soulbit mentioned. We will make a task to review the graph schema for typos

Reply:
I was looking at the Loan schema on the graph and I see `closewithDeposits` there but the type is `CloseWithDeposit`. Anyway, thanks for looking into that :)
<img width="356" alt="image" src="https://user-images.githubusercontent.com/14878713/174734241-d93c5926-7841-412c-95be-82a128ecc423.png">

Reply:
Hey Betsy!  i'm a bit confused), how the typo can be here? because if i change it to the "closeWithDeposit" - i will get the error message: "Type `Loan` has no field `closeWithDeposit`" and the same with extra "s"
the loan currently have "closewithDeposits" field

Reply:
Ah you guys are totally right!! Sorry - I will correct it

Reply:
I was looking at the data type instead of the field on Loan